posts/zh-Hans/2023-04-04-23.0.0.3.adoc

```diff
@@ -0,0 +1,626 @@
+我们花了一些时间来修复bug。下面的部分描述了在这个版本中解决的一些问题。如果您感兴趣，这里是link:https://github.com/OpenLiberty/open-liberty/issues?q=label%3Arelease%3A23003+label%3A%22release+bug%22[ 23.0.0.3中修复的完整错误列表]。
+
+
+
+//* link:https://github.com/OpenLiberty/open-liberty/issues/24566[AcmeCA feature with revocation enabled can fail to initialize on certain OS and JDK combinations]
+* link:https://github.com/OpenLiberty/open-liberty/issues/24566[启用了撤销的AcmeCA特性在某些操作系统和JDK组合上可能无法初始化]
+
++
+//When running with a hybrid JDK for MacOS with IBMJDK8 and  the Automatic Certificate Management Environment (ACME) Support 2.0 feature with certification revocation checking enabled, the SSL/TLS endpoint can fail to complete initialization and will not be available for traffic.
+当在MacOS上运行带有IBM JDK8和自动证书管理环境(ACME)支持2.0功能并启用证书撤销检查的混合JDK时，SSL/TLS端点可能无法完成初始化，并且无法用于通信。
++
+//The following `NullPointerException` is logged in the FFDC, indicating the failure in the initialization flow:
+FFDC里记录如下NullPointerException，表示初始化流程失败:
++
+[source]
+----
+Exception = java.lang.NullPointerException
+Source = com.ibm.ws.security.acme.internal.AcmeProviderImpl
+probeid = 921
+Stack Dump = java.lang.NullPointerException
+at sun.security.provider.certpath.CertPathHelper.setDateAndTime(CertPathHelper.java:71)
+at sun.security.provider.certpath.RevocationChecker.checkCRLs(RevocationChecker.java:525)
+at sun.security.provider.certpath.RevocationChecker.checkCRLs(RevocationChecker.java:464)
+at sun.security.provider.certpath.RevocationChecker.check(RevocationChecker.java:393)
+at sun.security.provider.certpath.RevocationChecker.check(RevocationChecker.java:336)
+at sun.security.provider.certpath.PKIXMasterCertPathValidator.validate(PKIXMasterCertPathValidator.java:125)
+at sun.security.provider.certpath.PKIXCertPathValidator.validate(PKIXCertPathValidator.java:225)
+at sun.security.provider.certpath.PKIXCertPathValidator.validate(PKIXCertPathValidator.java:145)
+at sun.security.provider.certpath.PKIXCertPathValidator.engineValidate(PKIXCertPathValidator.java:84)
+at java.security.cert.CertPathValidator.validate(CertPathValidator.java:304)
+at com.ibm.ws.security.acme.internal.CertificateRevocationChecker.isRevoked(CertificateRevocationChecker.java:371)
+....
+----
++
+//This issue has been resolved and the endpoint successfully initializes and is available to service traffic.
+此问题已解决，端点已成功初始化，并可用于服务通信。
+
+//* link:https://github.com/OpenLiberty/open-liberty/issues/24631[Fix ClassCastException during the de-serialization of CDI Injected Event]
+* link:https://github.com/OpenLiberty/open-liberty/issues/24631[修复CDI注入事件反序列化期间的ClassCastException问题]
++
+A passivated (i.e. serializable) bean which has an injected `jakarta.enterprise.event.Event` (or `javax` equivalent) will not be properly restored.  This can be encountered when session persistence is enabled and session data is serialized and de-serialized from a database.  This causes the following FFDC to occur:

```

Comment:
```suggestion
//A passivated (i.e. serializable) bean which has an injected `jakarta.enterprise.event.Event` (or `javax` equivalent) will not be properly restored.  This can be encountered when session persistence is enabled and session data is serialized and de-serialized from a database.  This causes the following FFDC to occur:

```